orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -122,6 +124,26 @@ def resume_process_endpoint(
     resume_process(process, user=user_name, user_inputs=json_data, broadcast_func=broadcast_func)
 
 
+@router.post("/{process_id}/continue/{token}", response_model=None, status_code=HTTPStatus.OK)
```

Comment:
What about just calling it `/{process_id}/callback/{token}`?

Reply:
But I don't have a problem with the endpoint name as it is. "Resume suspended" versus "Continue awaiting" is distinct enough

Reply:
callback in the url sounds good